pkg/ccn-lite/Makefile

```diff
@@ -7,8 +7,17 @@ PKG_LICENSE=ISC
 
 export RIOT_CFLAGS = ${CFLAGS} ${INCLUDES}
 
-all: git-download
-	cd $(PKG_BUILDDIR)/src && cmake -DCCNL_RIOT=1 -DRIOT_CFLAGS="${RIOT_CFLAGS}" . && make
+all: $(PKG_BUILDDIR)/src/Makefile
+	$(MAKE) -C $(PKG_BUILDDIR)/src && \
 	cp $(PKG_BUILDDIR)/src/lib/libccnl-riot.a ${BINDIR}/ccn-lite.a
 
+$(PKG_BUILDDIR)/src/Makefile: $(PKG_BUILDDIR)/xcompile-toolchain.cmake
```

Comment:
Please use a variable for `$(PKG_BUILDDIR)/xcompile-toolchain.cmake` its repeated 3 times in the build.

Reply:
good point